### PR TITLE
remove security vulnerabilities

### PR DESCRIPTION
**Pull request by jottenlips**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,13 +27,12 @@
...
     "express": "^4.16.4",
-    "jest": "^22",
-    "publish": "^0.6.0",
+    "jest": "^24.8.0",
     "ramda": "^0.25.0",
     "react": "16.2.0",
     "react-dom": "16.2.0",
     "react-functional-lifecycle": "^3.0.0",
-    "react-native": "0.50.0",
+    "react-native": "^0.60.0",
```
Comment (jottenlips) [inline package.json]: Upgrading RN was not causing any issues for me, might try installing it in a RN project. Also, is the new focus of this repo just for progressive web apps? Should probably get rid of the caret

File: package.json [hunk 2]
```diff
@@ -27,13 +27,12 @@
   "devDependencies": {
     "@types/jest": "^22",
     "express": "^4.16.4",
-    "jest": "^22",
-    "publish": "^0.6.0",
+    "jest": "^24.8.0",
```
Comment (jottenlips) [inline package.json]: had the most vulnerabilities

File: package.json [hunk 3]
```diff
@@ -27,13 +27,12 @@
   "devDependencies": {
     "@types/jest": "^22",
     "express": "^4.16.4",
-    "jest": "^22",
-    "publish": "^0.6.0",
```
Comment (jottenlips) [inline package.json]: was no longer being maintained

File: package.json (hunk 1 above)
Comment (tysoncadenhead) [reply package.json]: I think ultimately, I'd love for it to work on React Native and web if we're going to have other people using it outside of Aloompa, but we're only using it on the web right now.

File: package.json (hunk 2 above)
Comment (tysoncadenhead) [reply package.json]: Are we not using publish anywhere?

Comment (tysoncadenhead) [reply package.json]: oh, I see, you're patching and then publishing

Comment (benminer) [reply package.json]: ^

File: package.json (hunk 1 above)
Comment (benminer) [reply package.json]: @tysoncadenhead that would take some testing. Have we used this in a RN app before?

**Comment by tysoncadenhead** [comment]
👍 looks good to me

File: package.json (hunk 2 above)
Comment (jottenlips) [reply package.json]: most of the publish libraries seemed to not be maintained, I figure that would be good enough. and If we need to major we can manually change the package.json in a PR.

**Pull request merged**